eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -258,30 +258,34 @@ certificate =
   // stake_delegation
   // pool_registration
   // pool_retirement
-  // constitutional_deleg_cert
   // reg_cert
   // unreg_cert
   // vote_deleg_cert
   // stake_vote_deleg_cert
   // stake_reg_deleg_cert
   // vote_reg_deleg_cert
   // stake_vote_reg_deleg_cert
+  // reg_committee_cert
+  // unreg_committee_cert
   ]
 
 stake_registration = (0, stake_credential)
 stake_deregistration = (1, stake_credential)
 stake_delegation = (2, stake_credential, pool_keyhash)
 pool_registration = (3, pool_params)
 pool_retirement = (4, pool_keyhash, epoch)
-constitutional_deleg_cert = (5, genesishash, genesis_delegate_hash, vrf_keyhash)
-; number 6 used to be the MIR certificate, which is deprecated in Conway
+; numbers 5 and 6 used to be the Genesis and MIR certificates respectively,
+; which were deprecated in Conway
 reg_cert = (7, stake_credential, coin)
 unreg_cert = (8, stake_credential, coin)
 vote_deleg_cert = (9, stake_credential, voting_credential)
 stake_vote_deleg_cert = (10, stake_credential, pool_keyhash, voting_credential)
 stake_reg_deleg_cert = (11, stake_credential, pool_keyhash, coin)
 vote_reg_deleg_cert = (12, stake_credential, voting_credential, coin)
 stake_vote_reg_deleg_cert = (13, stake_credential, pool_keyhash, voting_credential, coin)
+reg_committee_cert = (14, credential, addr_keyhash)
```

Comment:
We can double check with @WhatisRT about his expectations, but I do not think we want a registration cert for the constitutional committee. The come into existence by a governance action. Unregistering, however, does make sense, we want the members to be able to step down at will.

Reply:
As far as I understand this, it is registration of the hot key, not the constitutional committee.

So, you can register and unregister the hot key, hence two certificates.

In any case semantics of the types will be defined by the rules, these are just types, which are translated from Agda spec and is called `ccreghot`:
https://github.com/input-output-hk/formal-ledger-specifications/blob/fb3722bfc945b5a2711fbc247f9e54fc73d4f326/src/Ledger/Deleg.lagda#L43

Let me know if you can suggest better names, maybe that is what makes this misleading?

Reply:
ah! yes, makes total sense new. I do like `reg_committee_hot_key_cert`, that's nice and clear.